contract_manager/src/contracts/evm.ts

```diff
@@ -541,6 +566,43 @@ export class EvmEntropyContract extends Storable {
     return web3.utils.randomHex(32);
   }
 
+  /**
```

Comment:
this is copy paste from the other class. You can create another BaseEvmContract class and put these shared logics there or just have some utility functions